cli: brush up `debug check-store`

tbg opened the pull request:
Check more Raft invariants, remove spurious errors, and improve the
output and exit code (which is now zero iff everything went well).

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/26517)
<!-- Reviewable:end -->

tbg committed:
cli: brush up `debug check-store`

Check more Raft invariants, remove spurious errors, and improve the
output and exit code (which is now zero iff everything went well).

Release note: None

bdarnell commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/26517#-:-LEQiHV81iwBoK35johQ:bnfp4nl)*
<!-- Sent from Reviewable.io -->

tschottdorf commented:
TFTR!

bors r=bdarnell

On Thu, Jun 7, 2018 at 3:29 PM Ben Darnell <notifications@github.com> wrote:

> [image: :lgtm:]
> <https://camo.githubusercontent.com/41b3ec3419116e3b3f84507ad965646ce4ce1f0c/68747470733a2f2f72657669657761626c652e696f2f6c67746d2e706e67>
> ------------------------------
>
> Review status: 0 of 1 files reviewed at latest revision, all discussions
> resolved, all commit checks successful.
> ------------------------------
>
> *Comments from Reviewable
> <https://reviewable.io/reviews/cockroachdb/cockroach/26517#-:-LEQiHV81iwBoK35johQ:bnfp4nl>*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/26517#issuecomment-395537553>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135JN79SiQHITSA1340nSOsHbEwyBzks5t6X8HgaJpZM4Ue3PC>
> .
>
-- 

-- Tobias
